backend/src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/service/VariablesService.java

```diff
@@ -46,11 +46,6 @@ public class VariablesService {
      * defined in the DataInjectionCommandLineRunner
      */
     private String demoRole;
-    @Value("${puris.apiversion}")
```

Comment:
Does it overall make sense to move the VariableService out of the common.api package into something like common.util?

Reply:
Yes, I agree. I'd suggest the same for the PatternStore